lib/index.js

```diff
@@ -15,11 +15,21 @@ const conditionalTranslate = (key, t) => {
   return result;
 };
 
-const getFields = (key, t) => ({
+const changeValidate = (optional) => {
+  let validateItems = ['required', 'postcode'];
+
+  if (optional) {
+    validateItems = ['postcode'];
+  }
+  return validateItems;
+};
+
+const getFields = (key, t, validator) => ({
+
   [`${key}-postcode`]: {
     label: conditionalTranslate(`fields.${key}-postcode.label`, t) || defaults.POSTCODE_LABEL,
     mixin: 'input-text-code',
-    validate: ['required', 'postcode'],
+    validate: validator,
```

Comment:
I'd just pass the `optional` flag in, and `optional ? ['postcode'] : ['required', 'postcode']`